base/poco/Foundation/include/Poco/Types.h

```diff
@@ -46,7 +46,6 @@ typedef unsigned long UInt64;
 typedef signed long long Int64;
 typedef unsigned long long UInt64;
 #        endif
-#    define POCO_HAVE_INT64 1
 #endif
```

Comment:
do we need `#if defined(__GNUC__) || defined(__clang__)` above?

Reply:
Not really, and I think there is a ton more dead code in poco.
- `__GNUC__` is support for the GNU C extensions (--> GCC and Clang). I think we should not rely on these.
- `__clang__` ... I don't remember if I removed that already in some other PR. I used [unifdef](https://dotat.at/prog/unifdef/) which does not always work correctly when a macro is part of an expression like here.

Anyways, this is not a big priority. I'll maybe push another PR.